src/react/App.js

```diff
@@ -3,12 +3,12 @@ import './App.css';
 import './Colors.css';
 
 // dummy data
-import dummyCompetitions from "../assets/mock-data/competitions.mock.data";
+import dummyCompetitions from '../assets/mock-data/competitions.mock.data';
```

Comment:
Kannst du mal schauen warum bei dir strings als `'string'` formatiert werden? Damit das nicht immer als Änderung auftaucht sollten es `"so"` sein :)

Reply:
ich find die einstellung nicht, nur die dass alles mit "" geschrieben wird

Reply:
ich hab das in einem späteren PR gefixt 👍